R/query.R

```diff
@@ -46,12 +47,13 @@ query.default <- function(connection, type, dataset, query, ...) {
 #' @export
 query.data.world <- function(connection, type = "sql", dataset, query, ...) {
```

Comment:
cool . will follow up with @bryonjacob on what parameterized queries mean ( not super familiar with the query api )  and add that in a separate PR